apps/ejabberd/src/ejabberd_c2s.erl

```diff
@@ -2868,7 +2875,7 @@ maybe_resume_session(NextState, El, StateData) ->
 do_resume_session(SMID, El, [{_, Pid}], StateData) ->
     try
         {ok, OldState} = ?GEN_FSM:sync_send_event(Pid, resume),
-        SID = {now(), self()},
+        SID = {os:timestamp(), self()},
```

Comment:
From repository perspective and organizing commits in "small" and "atomic" changes I suggest putting all the `now()` to `os:timestamp()` transitions in a separate commit.